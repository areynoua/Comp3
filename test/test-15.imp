begin
a := 1;
function @foo(a,b) do
	print(a);print(b);a := a + 1;c := c + 1;
	function @bar(b) do
		print(a);print(b);print(c)
	end;
	@bar(c)
end;
b := 2;
c := 3;
@bar(c);
@foo(a, 5);
@foo(6, 7);
print(a);
print(b);
print(c)
end
