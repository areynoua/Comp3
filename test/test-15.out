Unexpected symbol:
'function' with TOS: '<FuncInstList>'
LL1Parser:
    token number: 35
    stack: [ '<FuncInstList>', 'end', '<InstList-Tail>', 'end', '$' ]
    token: function	lexical unit: FUNCTION
    line: 5, column: 1
    code: + 1 ; function @bar ( b 

